Sourabh_Kanwade/html/Assignment 2/Question_1.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Html Formating</title>
+  </head>
+  <body style="background-color: #151515; color: #b4a5a5">
+    <center>
+      <h1>Google</h1>
+    </center>
+    <small> <i>This article is about the company.</i> </small>
+    <h2>From the garage to the Googleplex</h2>
+    The Google story begins in 1995 at <b>Stanford University</b>
+    <br />
+    <b> <i>Larry Page</i> </b>was considering Stanford for grad school and
+    <b><i>Sergey Brin,</i></b>
+    a student
+    <br />
+    there, was assigned to show him around.
+    <br />
```

Comment:
remove extra <br> opening and closing tag format code properly

Reply:
should I replace <br /> with <br> ?

Reply:
yes

Reply:
done